docs/items/index.md

```diff
@@ -1,72 +1,127 @@
-| `requiredFeatures` | Sets the required `FeatureFlag`s needed to see this item in the `CreativeModeTab` it is added to. |
-| `durability`       | Sets the maximum damage value for this item. If it is over `0`, two item properties "damaged" and "damage" are added. |
-| `stacksTo`         | Sets the maximum stack size. You cannot have an item that is both damageable and stackable. |
-| `setNoRepair`      | Makes this item impossible to repair, even if it is damageable. |
-| `craftRemainder`   | Sets this item's container item, the way that lava buckets give you back an empty bucket when they are used. |
+## Creating an Item
 
-The above methods are chainable, meaning they `return this` to facilitate calling them in series.
+Now that we understand what an item is, let's create one!
 
-### Advanced Items
+Like with basic blocks, for basic items that need no special functionality (think sticks, sugar, etc.), the `Item` class can be used directly. To do so, during registration, instantiate `Item` with a `Item.Properties` parameter. This `Item.Properties` parameter can be created using `Item.Properties#of`, and it can be customized by calling its methods:
 
-Setting the properties of an item as above only works for simple items. If you want more complicated items, you should subclass `Item` and override its methods.
+- `stacksTo` - Sets the max stack size of this item. Defaults to 64. Used e.g. by ender pearls or other items that only stack to 16.
+- `durability` - Sets the durability of this item. Defaults to 0, which means "no durability". For example, iron tools use 250 here. Note that setting the durability automatically locks the stack size to 1.
+- `craftRemainder` - Sets the crafting remainder of this item. Vanilla uses this for filled buckets that leave behind empty buckets after crafting.
+- `fireResistant` - Makes item entities that use this item immune to fire and lava. Used by various netherite items.
+- `setNoRepair` - Disables anvil and crafting grid repairing for this item. Unused in vanilla.
+- `rarity` - Sets the rarity of this item. Currently, this simply changes the item's color. `Rarity` is an enum consisting of the four values `COMMON` (white, default), `UNCOMMON` (yellow), `RARE` (aqua) and `EPIC` (light purple). Be aware that mods may add more rarity types.
+- `requiredFeatures` - Sets the required feature flags for this item. This is mainly used for vanilla's feature locking system in minor versions. It is discouraged to use this, unless you're integrating with a system locked behind feature flags by vanilla.
+- `food` - Sets the [`FoodProperties`][food] of this item.
+
+For examples, or to look at the various values used by Minecraft, have a look at the `Items` class.
+
+### Food
+
+The `Item` class provides default functionality for food items, meaning you don't need a separate class for that. To make your item edible, all you need to do is set the `FoodProperties` on it through the `food` method in `Item.Properties`.
```

Comment:
It might be useful to point out the methods to determine whether an item is considered food or how food changes the default behavior.